types/pino/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "dependencies": {
+    "sonic-boom": "^2.0.1"
```

Comment:
Sorry, but this doesn't fly. Pino still uses `sonic-boom@1`. You'll need to depend on `@types/sonic-boom@1` instead.

Reply:
@BendingBender We are likely to merge the change to use newer sonic-boom soon: https://github.com/pinojs/pino/pull/1039
The whole reason for introduction of 2.0.0 was the embedding of TS types.

Reply:
Yeah well, you'll have to wait until a version of pino is released that depends on `sonic-boom@2` and bump it here then. Until then, you should depend on  `@types/sonic-boom@1` as far as I understand it. That shouldn't be an issue, you can add an exclusion for this in `DefinitelyTyped-tools`, the same way you made it for `sonic-boom`.

Reply:
Having thought a bit more about it, I probably would suggest removing pino changes out of this PR altogether, and just deprecate pino typings altogether after next semver major comes out, since that will also have embedded types.

Reply:
You can't remove `sonic-boom` types without explicitly depending on `@types/sonic-boom` for `pino`'s types. You can try it out and see the lint issue you'll get. So the way forward would be to:

- add `@types/sonic-boom` to the allowlist in `DefinitelyTyped-tools`
- make `pino`'s types explicitly depend on `@types/sonic-boom`
- add `sonic-boom` to `notNeededPackages.json` and remove the types